fortitude/src/cli.rs

```diff
@@ -165,6 +165,21 @@ pub struct CheckArgs {
     #[arg(long, value_enum)]
     pub progress_bar: Option<ProgressBar>,
 
+    /// See the settings fortitude will use to check a given Fortran file.
+    #[arg(long,
+        // Fake subcommands.
+        conflicts_with = "show_files",
+        // conflicts_with = "show_settings",
+    )]
+    pub show_settings: bool,
```

Comment:
I'm a bit confused with the commented out lines here.

Reply:
There are others for the `show_files` setting too.

Reply:
It's your call. I'd be happy to merge them here, but it might be better for the changelog to add them separately.

Reply:
I'm just going to merge this now, and we'll have to remember to manually add those flags to the changelog!